services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/ApplicationConfig.java

```diff
@@ -87,4 +90,31 @@ public RegistrationAssertionHelper registrationAssertionFactory() {
         return RegistrationAssertionHelperImpl.forSigning(vertx(), deviceRegistryProps.getRegistrationAssertion());
     }
 
+    /**
+     * Expose Hono's <a href="https://www.eclipse.org/hono/api/Device-Registration-API/">Device Registration API</a> endpoint as a Spring bean.
+     * <p>
+     * See {@link RegistrationEndpoint} for more details.
+     */
+    @Bean
+    @Qualifier("registration")
```

Comment:
Actually, I asked myself the same question. But it was there before at the @Component annotation, so I rather kept it and moved it here.
I will remove it since it might be historical and without use now - WDYT?